Assets/Scripts/LoadMainMenu.cs

```diff
@@ -0,0 +1,10 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class LoadMainMenu : MonoBehaviour
+{
+    public void OnClick()
+    {
+        SceneManager.LoadScene(1);
```

Comment:
```suggestion
        PlayerPrefs.Save();
        SceneManager.LoadScene(1);
```
Would you be able to update this to save the player prefs before moving onto the Main Menu?
This would allow your subsequent `GetFloat` calls to stick as the preferences would be persisted across scenes